main/templates/main/landing_page.html

```diff
@@ -3,11 +3,22 @@
 <html>
 <head>
     <title>roo.me - Start now!</title>
+    <!-- Bootstrap core CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```

Comment:
I think we already have this library included in this file.

Reply:
Cant find it in this file, can you check again?

Reply:
![Screen Shot 2021-05-19 at 14 55 56](https://user-images.githubusercontent.com/53123142/118808607-5e41eb80-b8b2-11eb-81cf-e99ee07afab3.png)

Did you pull from upstream?

Reply:
Oh this is the landing page. My mistake, I thought it was the base template for the home page.